content/en/docs/instrumentation/python/automatic/operator.md

```diff
@@ -0,0 +1,121 @@
+
+This can be done directly via the [Operator release manifest](https://github.com/open-telemetry/opentelemetry-operator#getting-started), via the [Operator helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator#opentelemetry-operator-helm-chart), or via [Operator Hub](https://operatorhub.io/operator/opentelemetry-operator).  
+
+In most cases you will need to the [cert-manager installed](https://cert-manager.io/docs/installation/), but if using the helm chart there is an option to generate a self-signed cert.
+
+## Create an OpenTelemetry Collector (Optional)
+
+It is a best practice to send telemetry from containers to an [OpenTelemetry Collector](../../../../collector/) instead of directly to a backend.  The Collector helps simplify secret management, provides reliability and scalability, and can help improve data (k8sattributes processor).  If you chose not to use a Collector you can skip to the next section.
+
+The Operator provides a [Custom Resource Definition (CRD) for an OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#opentelemetrycollector) which is used to create an instance of the Collector that the Operator will manage. This example will deploy the Collector as a deployment, which is the default, but there are other [deployment modes](https://github.com/open-telemetry/opentelemetry-operator#deployment-modes) that can be used.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: demo
+spec:
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 75
+        spike_limit_percentage: 15
+      batch:
+        send_batch_size: 10000
+        timeout: 10s
+
+    exporters:
+      logging:
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [memory_limiter]
+          exporters: [logging]
+        metrics:
+          receivers: [otlp]
+          processors: [memory_limiter]
+          exporters: [logging]
+EOF
+```
+This creates an deployment of the Collector that we can use as an endpoint for the auto-instrumentation.
+
+## Configure the Instrumentation
+
+In order to auto-instrument services the operator needs to be told how and what to instrument.  This is done via the [Instrumenation CRD](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentation).
+
+Creating the Instrumentation object correctly is paramount to getting auto-instrumentation working.  Via the instrumentation object you are  configuring the OpenTelemetry SDK for python. Making sure all endpoints and env vars are correct is required for auto-instrumentation to work properly.
```

Comment:
I don't think I understand how this section relates to the code sample below. Where does it specify that we're using the Python SDK?

Reply:
This paragraph is supposed to be a warning that configuring the Instrumentation resource is both the trickiest but most important step of injecting auto-instrumentation.  I was trying to highlight that some of the items set in the Instrumentation resource map to items in the SDK that the operator configures and then inject.  

When I was learning this process I got held up for a second realizing that setting all the env vars in the Instrumentation object was the same as setting all the env var for an application that implemented the SDK itself.  In order to correctly configure the Instrumentation resource you have to understand the env vars that the SDK uses/depend on.

How does this rewording sound?

```suggestion
Creating the Instrumentation resource correctly is paramount to getting auto-instrumentation working.  Making sure all endpoints and env vars are correct is required for auto-instrumentation to work properly.
```

Reply:
Yep, I like it.